azure.gemspec

```diff
@@ -50,3 +49,4 @@ Gem::Specification.new do |s|
   s.add_development_dependency('yard',                '~> 0.8')
   s.add_development_dependency('vcr',                 '~> 3.0')
 end
+
```

Comment:
I think we left our conversation half-way (From: https://github.com/Azure-Samples/compute-ruby-manage-vm/pull/3) regarding this one. I believe that we should follow coding standards of `Rails` & `Ruby` by only having `CRLF` character at the end of line instead a new line because `rubocop` will consider this as offense. 

What are your thoughts?